### DISCUSSION TITLE
Problem installing lokinet

### DISCUSSION THREAD

**Discussion by Bar2ly** [oxen-io/lokinet#1814] [Q&A] [answered]
I want to install lokinet in linux and i just compiled it. When i execute "lokinet" in the command line it gives me an error like this: 

[NFO] [lokinet](515) 2021-11-28 18:48:52.252 GMT [+0.000s] /daemon/lokinet.cpp:238Using config file: "/home/nacho/.lokinet/lokinet.ini"
[NFO] [lokinet](515) 2021-11-28 18:48:52.253 GMT [+0.001s] /llarp/context.cpp:64	lokinet-0.9.8-b23ceefc (dev build)
[NFO] [lokinet](515) 2021-11-28 18:48:52.253 GMT [+0.001s] /llarp/context.cpp:65	starting up
[NFO] [lokinet](515) 2021-11-28 18:48:52.253 GMT [+0.001s] /llarp/router/router.cpp:1118	Bound RPC server to tcp://127.0.0.1:1190
[ERR] [lokinet](515) 2021-11-28 18:48:52.254 GMT [+0.002s] /llarp/router/router.cpp:594	No bootstrap files specified in config file, and the default
[ERR] [lokinet](515) 2021-11-28 18:48:52.254 GMT [+0.002s] /llarp/router/router.cpp:595	bootstrap file "/home/nacho/.lokinet/bootstrap.signed" does not exist.
[ERR] [lokinet](515) 2021-11-28 18:48:52.254 GMT [+0.002s] /llarp/router/router.cpp:596	Please provide a bootstrap file (e.g. run 'lokinet-bootstrap)'
[ERR] [lokinet](515) 2021-11-28 18:48:52.254 GMT [+0.002s] /daemon/lokinet.cpp:269failed to set up lokinet: No bootstrap files available., is lokinet already running? 🤔

I suppose the error is that there is no bootstrap.signed file in the ~.lokinet directory, i made one but I suppose there must be something in there and it gave me an error like this: 

[NFO] [lokinet](534) 2021-11-28 18:41:14.021 GMT [+0.017s] /daemon/lokinet.cpp:238Using config file: "/home/nacho/.lokinet/lokinet.ini"
[NFO] [lokinet](534) 2021-11-28 18:41:14.022 GMT [+0.018s] /llarp/context.cpp:64	lokinet-0.9.8-b23ceefc (dev build)
[NFO] [lokinet](534) 2021-11-28 18:41:14.030 GMT [+0.026s] /llarp/context.cpp:65	starting up
[NFO] [lokinet](534) 2021-11-28 18:41:14.035 GMT [+0.031s] /llarp/router/router.cpp:1118	Bound RPC server to tcp://127.0.0.1:1190
[ERR] [lokinet](534) 2021-11-28 18:41:14.037 GMT [+0.033s] /daemon/lokinet.cpp:269failed to set up lokinet: failed to decode bootstrap RC, file='"/home/nacho/.lokinet/bootstrap.signed"' rc=[ k = [ "0000000000000000000000000000000000000000000000000000000000000000" ] updated = 0 netid = [ "lokinet" ] v = 0 ai = [ ] e = [ "0000000000000000000000000000000000000000000000000000000000000000" ] z = [ "00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000" ] ], is lokinet already running? 🤔

help?

**Comment by majestrate** [comment, marked as answer]
yeah, you'll want to copy `contrib/bootstrap/mainnet.signed` to `~/.lokinet/bootstrap.signed`

**Reply by Bar2ly** [reply]
Worked. Now that I run lokinet am I already using lokinet? there is no gui where I can configure things?

**Reply by majestrate** [reply]
there is a lokinet.ini at `~/.lokinet/lokinet.ini` with basically all of the configurable options. you can override them by placing .ini files into `~/.lokinet/conf.d/` and they will be loaded in lexicographical order. you dont need to edit lokinet.ini if you dont want to and i often find using the `conf.d` directory to be a lot less messy.
